vnext/Microsoft.ReactNative.Cxx/CppWinRTIncludes.h

```diff
@@ -13,6 +12,7 @@
 #include <winrt/Windows.System.h>
 #include <winrt/Windows.UI.Xaml.h>
 
+#define XAML_CPPWINRT_NAMESPACE winrt::Windows::UI::Xaml
 namespace xaml = winrt::Windows::UI::Xaml;
```

Comment:
For my education: Is this header file specifically meant for Xaml-based consumers?

Reply:
Yes, this is for XAML users. We are going to split up the project to separate the "core" files from XAML. We just need to change some ABI files first.

---
In reply to: [433449630](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r433449630) [](ancestors = 433449630)